test/markup/fortran/numbers.txt

```diff
@@ -12,4 +12,20 @@
 1D-4
 var1
 va1r
-mo_tot_8 = 1./(0.4*log(float(elec_num_tot_8+0.4)))
\ No newline at end of file
+mo_tot_8 = 1./(0.4*log(float(elec_num_tot_8+0.4)))
+
+CALL testing(6_ikind, 4, 2)
+CALL testing(4_ikind, 4, 1)
+CALL testing(4_ikind, 2, -1)
+CALL testing(35_ikind, 7, 3)
+CALL testing(36_ikind, 7, -1)
+CALL testing(184756_ikind, 20, 10)
+CALL testing(184756_ikind, 10, -1)
+CALL testing(3268760_ikind, 25, 10)
+CALL testing(155117520_ikind, 30, 15)
+CALL testing(155117530_ikind, 30, -1)
+CALL testing(6540715896_ikind, 48, 10)
+CALL testing(6540715897_ikind, 48, -1)
+CALL testing(344867425584_ikind, 45, 15)
+CALL testing(118264581564861424_ikind, 60, 30)
+CALL testing(53194089192720_ikind, 60, 15)
```

Comment:
I don't think we need all of these. For Fortran number literals test, more different formats are needed. But here you want to test the parser bug, so maybe a separate test in `test/parser` would be better?

Reply:
Ok you want a test that was red before and green now, that makes sense. I'll see what I can do.

I'll remove a few of these and see if I can add a little more variety from the other issue, not super worried about this though.  It can be improved when the actual issue is fixed.  So technically it could be removed here, but I like getting it committed.